src/components/Disputes/DisputeInfoContent.js

```diff
@@ -296,7 +298,10 @@ const DisputedActionNA = () => {
       >
         N/A
       </span>
-      <Help hint="">The disputed Action</Help>
+      <Help hint="">
+        This dispute is simply between the two arguments. There's no executable
+        action involved.
+      </Help>
```

Comment:
Not sure if this is enough information for the user or if it's formal enough. Suggestions are welcome

Reply:
Could you leave a screenshot for this case? I was looking at the deployed version to Staging, but couldn't find a dispute like this (should dispute 26 show this?).

Reply:
Hmm, it should show properly
 
![Screen Shot 2020-08-07 at 3 08 51 PM](https://user-images.githubusercontent.com/22663232/89675307-15d7b500-d8c0-11ea-82a1-38702d6da340.png)